samples/BenchmarkDotNet.Samples/BenchmarkDotNet.Samples.csproj

```diff
@@ -11,6 +11,7 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DebugSymbols>true</DebugSymbols>
     <NoWarn>$(NoWarn);CA1018;CA5351;CA1825</NoWarn>
+    <GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
Yes, it is mandatory. Without this, when the Test SDK is referenced, it will automatically copy into the project a default entry point and the compilation will fail.

Having said this, my other PR solves this exact issue and provides a way for this property to be set just by referencing the NuGet Package, so when this PR is merged in we no longer need this property to be set in the csproj: https://github.com/dotnet/BenchmarkDotNet/pull/2445